freqtrade/exchange/exchange.py

```diff
@@ -656,13 +661,7 @@ def get_trades_for_order(self, order_id: str, pair: str, since: datetime) -> Lis
 
     @retrier
     def get_markets(self) -> List[dict]:
-        try:
-            return self._api.fetch_markets()
-        except (ccxt.NetworkError, ccxt.ExchangeError) as e:
-            raise TemporaryError(
-                f'Could not load markets due to {e.__class__.__name__}. Message: {e}')
-        except ccxt.BaseError as e:
-            raise OperationalException(e)
+        return list(self.markets.values())
```

Comment:
a few points on this:

why don't you remove `get_markets()` completely?
It makes no sense to me if it's not doing anything differently than what I can get with exchange.markets.
Also, this function is ONLY used in tests (7 times, yes I did count ...)  - nowhere in "real" code.

If you think it needs to stay:
please remove `@retrier` - there's no point in calling this 3 times, it either fails or works.

Reply:
I was thinking about removing it, but I kept it for backward compatibility. I don't know if there are many/any projects that build on FT, but they might be using `get_markets`. Other than that, I agree it can go.

Reply:
Honestly, I don't really see freqtrade as a library / upstream dependency, but as a standalone bot. (i'm also not aware of any downstream projects other than forks ...)

I would currently not know many benefits the exchange class would give you over using ccxt directly outside of the bot's context.

Reply:
Completely agree, just didn't want to delete it right away. If you say it's unnecessary, I'll take it out.